examples/resource_lacework_integration_aws_org_agentless_scanning/main.tf

```diff
@@ -6,6 +6,11 @@ terraform {
   }
 }
 
+provider "lacework" {
+  alias = "organization"
```

Comment:
This might be the problem, if you configure an alias to this provider, you have to use it on every resource, an example is:
```hcl
resource "lacework_integration_aws_org_agentless_scanning" "example" {
    provider = lacework.organization
    name = "foo"
    ...
}
```